commands/ft._list.md

```diff
@@ -0,0 +1,11 @@
+## FT._LIST
```

Comment:
The command name is automatically added, you can omit this. You can omit the back ticks as well.